client/store/plugins/vuex-persist.js

```diff
@@ -13,10 +13,17 @@ export default new VuexPersistence({
     auth: state.auth,
     repository: {
       comments: {
-        seenByActivity: state.repository.comments.seenByActivity
+        seenByActivity: state.repository.comments.seen.activity,
+        seen: migrateSeenState(state)
       }
     }
   }),
   storage: window.localStorage,
   filter: mutation => OBSERVED_MUTATIONS.includes(mutation.type)
 }).plugin;
+
+function migrateSeenState(state) {
```

Comment:
Consider rather checking upon boot if there is a key in local storage, change its location, and delete it.